src/main/java/org/opentripplanner/street/search/state/State.java

```diff
@@ -54,7 +56,7 @@ public class State implements AStarState<State, Edge, Vertex>, Cloneable {
   /**
    * Create an initial state, forcing vertex to the specified value. Useful for tests, etc.
    */
-  public State(Vertex vertex, StreetSearchRequest streetSearchRequest) {
+  public State(@Nonnull Vertex vertex, StreetSearchRequest streetSearchRequest) {
```

Comment:
I guess the streetSearchRequest should also be non null but I don't know if we need to mark it as such here.

Reply:
I've added the annotations in b9aeef1751093f6f1eb9ac184cd829c90110a8ac